### PR TITLE
Add editor form

### PR DESCRIPTION
**Pull request by chillilauren**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -61,6 +60,12 @@ app.get("/home", (request, response) => {
     response.render('index.html', model);
 });
 
+app.get("/admin/editors/new", async (request, response) => {
+    const model = {
+        message: "Admin Site"
+    }
```
Comment (MichaelWalker) [inline src/app.ts]: does this model do anything?
looks like we might be able to just get rid of it?

File: src/nasa/nasaApi.ts [hunk 2]
```diff
@@ -12,9 +12,6 @@ interface RoverImage {
     imgSource: string;
 }
 
-// interface PhotoData {
-//     imageSource: string
-// }
```
Comment (MichaelWalker) [inline src/nasa/nasaApi.ts:17]: nice - thanks for removing

File: templates/adminEditor.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<form method="post" action="/admin/editors/new" >
+    <p>Add new editor</p>
+    <label for="id">Email</label>
+    <input id="email" type="string" name="email" />
+
+    <label for="password">Password</label>
+    <input id="password" type="password" name="password" />
+
+    <button type="submit">Submit</button>
+</form>
```
Comment (MichaelWalker) [inline templates/adminEditor.html:14]: HTML looks good.
Is there any styling for this though?

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
What you've got looks really good, but also like there isn't any styling yet?
(and it looks like CircleCI is failing too)

Could you add a test to make the build pass?
After that I think we should merge this in, and perhaps create a new ticket for the styling?

File: templates/adminEditor.html (hunk 3 above)
Comment (chillilauren) [reply templates/adminEditor.html:14]: Are we supposed to add a scss file into the backend?

**Review by MichaelWalker — APPROVED** [review]

**Pull request merged**